config/kernel-group-info.m4

```diff
@@ -6,8 +6,8 @@ AC_DEFUN([ZFS_AC_KERNEL_SRC_GROUP_INFO_GID], [
 	ZFS_LINUX_TEST_SRC([group_info_gid], [
 		#include <linux/cred.h>
 	],[
-		struct group_info *gi = groups_alloc(1);
-		gi->gid[0] = KGIDT_INIT(0);
+		struct group_info gi = {};
```

Comment:
Seems like we might want `__attribute__ ((unused))` here to ward off any potential future compiler warnings. Seems like `gcc` isn't currently picking up on this.